application/applicationextension.inc.php

```diff
@@ -2269,4 +2269,46 @@ public function InitStats();
      * @return mixed
      */
     public function LogOperation($oKpiLogData);
+}
+
+/**
+ * Interface to provide messages to be displayed in the "Welcome Popup"
+ *
+ * @api
+ * @private
+ * @since 3.1.0
+ */
+interface iWelcomePopup
```

Comment:
```suggestion
interface iWelcomePopupExtension
```

Match naming convention of other APIs